storage/innobase/xtrabackup/test/run.sh

```diff
@@ -846,6 +860,7 @@ recognized for compatibility";;
 done
 
 set_vars
+config_local_components
```

Comment:
what is the condition for calling config_local_components?

Reply:
we call it regardless of any condition. If testing on a server prior to 8.0.24 this change won't take effect. If calling it on a server from 8.0.24 forward, the server will load the components (if configured by tests) from datadir instead of bin & plugin. This is called global config vs local config. It's necessary otherwise enabling the keyring component in one test would enable it on all tests running in parallel, since they share the same bin & plugin folders.

Reply:
Thanks for the detailed explanation!

Reply:
It is saving mysqld.my file in 8.0.23 server directory. not a big deal but better to to put condition if server_version greater 8.0.23.

Reply:
Done